### PR TITLE
Implemented Google Recaptcha

### PR DESCRIPTION
**Pull request by rashmi-91**

### CODE REVIEW

File: src/components/HomePage/index.js [hunk 1]
```diff
@@ -19,13 +19,14 @@ import feature8 from './assets/feature-8.svg';
...
 import { sendContactForm } from '../../api/sendContactForm';
-import { WCBCT } from './components/WCBCTbanner'
+import ReCAPTCHA from "react-google-recaptcha";
 
 export default class HomePage extends Component {
 
   constructor(props) {
     super(props);
   }
+  recaptchaRef = React.createRef();
```
Comment (ItamarGronich) [inline src/components/HomePage/index.js:29]: Where are you using this?
There's no `this.recaptchaRef ` in any place other than the Recaptcha component

File: src/components/HomePage/index.js [hunk 2]
```diff
@@ -164,14 +165,19 @@ export default class HomePage extends Component {
...
             <form className="contact-form" onSubmit={this.handleSubmit}>
               <input type="text" id="contact-form-name" name="name" placeholder="Name" className="contact-section-input margin-bottom-medium" required/>
               <input type="text" id="contact-form-country" name="country" placeholder="Country" className="contact-section-input margin-bottom-medium"/>
               <input type="text" id="contact-form-phone" name="phone" placeholder="Phone" className="contact-section-input margin-bottom-medium"/>
               <input type="email" id="contact-form-email" name="email" placeholder="Email" className="contact-section-input margin-bottom-medium" required/>
               <textarea rows="40" id="contact-form-message" name="message" placeholder="Message" className="contact-section-input margin-bottom-medium"></textarea>
+              <ReCAPTCHA
+                ref={this.recaptchaRef}
+                sitekey="6LdyBuQZAAAAAHZRoMHSxkXl9YSW8lNBUlOQCKVS"
+                onChange={this.onChange}
```
Comment (ItamarGronich) [inline src/components/HomePage/index.js:179]: How is this used?
I don't see any `onChange` function defined anywhere in the component.

File: src/components/HomePage/index.js [hunk 3]
```diff
@@ -164,14 +165,19 @@ export default class HomePage extends Component {
...
+              <p className="margin-bottom-large"><b>support@PsySession.com? <a className="tel" href="tel:+972-556600420">(+972)-55-660-0420</a></b></p>
             </div>
             <form className="contact-form" onSubmit={this.handleSubmit}>
               <input type="text" id="contact-form-name" name="name" placeholder="Name" className="contact-section-input margin-bottom-medium" required/>
               <input type="text" id="contact-form-country" name="country" placeholder="Country" className="contact-section-input margin-bottom-medium"/>
               <input type="text" id="contact-form-phone" name="phone" placeholder="Phone" className="contact-section-input margin-bottom-medium"/>
               <input type="email" id="contact-form-email" name="email" placeholder="Email" className="contact-section-input margin-bottom-medium" required/>
               <textarea rows="40" id="contact-form-message" name="message" placeholder="Message" className="contact-section-input margin-bottom-medium"></textarea>
+              <ReCAPTCHA
+                ref={this.recaptchaRef}
```
Comment (ItamarGronich) [inline src/components/HomePage/index.js:177]: Same as the the comment before i don't see `this.recaptchaRef` used anywhere.

**Review by ItamarGronich — APPROVED** [review]
Did you run the full flow once end-to-end and checked that it works?

File: src/components/HomePage/index.js (hunk 2 above)
Comment (rashmi-91) [reply src/components/HomePage/index.js:179]: onChange is not required. Removed it.

File: src/components/HomePage/index.js (hunk 1 above)
Comment (rashmi-91) [reply src/components/HomePage/index.js:29]: This is required for google recaptcha checkbox to get selected. Without this, checkbox is not actionable. 
So, used it.

**Comment by rashmi-91** [comment]
> Did you run the full flow once end-to-end and checked that it works?

Yes, it is working.

File: src/components/HomePage/index.js (hunk 3 above)
Comment (rashmi-91) [reply src/components/HomePage/index.js:177]: This is required for google recaptcha checkbox to get selected. Without this, checkbox is not actionable.
So, used it.

**Pull request merged**